applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -72,20 +72,23 @@ struct temp_cap_storage {
 	struct bt_audio_codec_cap codec[CONFIG_CODEC_CAP_COUNT_MAX];
 };
 
-static struct le_audio_unicast_server unicast_servers[CONFIG_BT_MAX_CONN];
+/* Since there is no subgroups for CIG we will use 1 as a hard coded value */
```

Comment:
Well explained. Shall we also do a define? E.g. #define CIG_SUBGROUP 1 ?

Reply:
We could, unless that gives the impression that CIGs can have multiple subgroups